apps/api/src/donations/donations.service.ts

```diff
@@ -41,6 +42,9 @@ export class DonationsService {
     private exportService: ExportService,
   ) {}
 
+  private cache = new HotCache()
+  private cacheTimeout = 30 * 1000 // 30 seconds
```

Comment:
could you make that an env variable? It would allow us to fine tune the value more easily. I was even thinking we can cache it for 1 or 2 mins. If it's exposed as a setting we can tweak it and see how it works

Reply:
Good idea.

Reply:
Doesn't sound very scalable. Lets say for example, later on we decide to cache the result from the query which fetches all active and completed campaigns. Since this result is not expected to be as regularly updated, as the platform statistics, a bigger expiration time could be set

Reply:
we now have a default ttl in the config, but each service and variable can define its own expiration time.